tests/test_sqlalchemy_data_layer.py

```diff
@@ -158,7 +158,7 @@ def engine(
         person_tag_model, person_single_tag_model, person_model,
         computer_model, string_json_attribute_person_model,
         address_model
-    ):
```

Comment:
Oops

Reply:
Do I need rollback this whitespace error ?
I use idea ultimate with automatic formatter so probably how its get there. Could by nice to have some formatter definition like flake8 or pylint.

Reply:
Well, no need
Yeah, we better add another pipeline / git hook with formatting